Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -1,24 +1,78 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BootCamp.Chapter
 {
     public class CredentialsManager
     {
         private readonly string _credentialsFile;
 
         public CredentialsManager(string credentialsFile)
         {
+            if (string.IsNullOrEmpty(credentialsFile))
+            {
+                throw new ArgumentException();
+            }
             _credentialsFile = credentialsFile;
         }
 
         // TODO: load credentials and check for equality.
         public bool Login(Credentials credentials)
         {
+            var existingCredentials = GetCredentials();
+
+            for (int i = 0; i < existingCredentials.Count; i++)
+            {
+                if (credentials.Equals(existingCredentials[i]))
+                {
+                    return true;
+                }
+            }
+
             return false;
         }
 
         // TODO: store credentials in credentials file.
         public void Register(Credentials credentials)
         {
+            if (File.ReadAllLines(_credentialsFile).Length > 0)
+            {
+                File.AppendAllText(_credentialsFile, Environment.NewLine);
+            }
+            
+            File.AppendAllText(_credentialsFile, credentials.ToFile);
+        }
+
+        List<Credentials> GetCredentials()
+        {
+            List<Credentials> creds = new List<Credentials>();
 
+            string fileContent = File.ReadAllText(_credentialsFile);
```

Comment:
You can use ``File.ReadAllLines()`` to read file as an array of lines right away.